db.json

```diff
@@ -4,22 +4,26 @@
       {
         "id": 1,
         "name": "architecture",
-        "description": "Some archy description."
+        "description": "Some archy description.",
+        "tooltip": "{proportion} of users have the {x} architecture"
```

Comment:
Talked with Rob on IRC about the latter. I was assuming all templates were the same:

> {x} of users have {y}

But if the template might change from chart to chart, I think it makes sense to have Brendan input the template through Django admin and expose that via the API.